unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Debugging/Performance/PerformanceMetricsController.cs

```diff
@@ -99,7 +99,7 @@ private void Report(string encodedSamples)
                 if (coords.x == 0 && coords.y == 0)
```

Comment:
Instead of the coordiantes, let's check with `parcelScene.isPersistent` to know if it's the global scene (as we do in [this other place](https://github.com/decentraland/unity-renderer/blob/7415d9f6526d1afa46c266b07af667a072f27b22/unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/Utils/PositionUtils/WorldStateUtils.cs#L16)